crates/storage/src/lazy/mapping.rs

```diff
@@ -161,13 +161,24 @@ where
     ///
     /// Returns `None` if no `value` exists at the given `key`.
     #[inline]
-    pub fn contains<Q>(&self, key: Q) -> Option<u32>
+    pub fn size<Q>(&self, key: Q) -> Option<u32>
     where
         Q: scale::EncodeLike<K>,
     {
         ink_env::contract_storage_contains(&self.storage_key(&key))
     }
 
+    /// Get the size of a value stored at `key` in the contract storage.
```

Comment:
Copy paste comment.

Reply:
I had modified it to reflect that it's the size, how do you think it should be instead?

Reply:
But this comment is on `contains`. Not on `size`.